dl4j-cv-labs/src/main/java/global/skymind/solution/object_detection/AvocadoBananaDetector/dataHelpers/NonMaxSuppression.java

```diff
@@ -0,0 +1,64 @@
+package global.skymind.solution.object_detection.AvocadoBananaDetector.dataHelpers;
+
+import org.deeplearning4j.nn.layers.objdetect.DetectedObject;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Created on Jun 21, 2018 , 4:26:26 PM
+ *
+ * @author Taha Emara
+ * Email : taha@emaraic.com
+ * Website: http://www.emaraic.com
+ * Implementation is based on code from here https://deeplearning4j.org/autonomous-vehicle-java
+ */
+public class NonMaxSuppression {
```

Comment:
repetitive class, can refactor it to a common folder?

Reply:
done, moved to object_detection/dataHelpers